### DISCUSSION TITLE
Double tap / drag without that stupid 0.5s delay

### DISCUSSION THREAD

**Discussion by skiv71** [rxhanson/Multitouch-Community#164] [General]
Hi, can this be done?

Coming from win / linux, this operarion is a must, but macos's over engineered implementation (and this timer) makes it hell

**Comment by rxhanson** [comment]
If I understand correctly, you are experiencing a 0.5 second delay when using the built-in macOS setting for this. For me, it's instantaneous, so you might have something unusual about your setup? 

Multitouch does have the "Left Click and Drag" action, but it does not have a "double tap anywhere" gesture. 

Since the macOS built-in three finger drag gesture for drag and drop can conflict with other gestures like three finger click for middle click, I typically recommend the "rest three, remove all but one" gesture for "Left Click and Drag", although I realize that's not what you're looking for here.
